src/main/webapp/index.html

```diff
@@ -3,72 +3,11 @@
-    <script src="https://unpkg.com/@chainsafe/libp2p-noise/dist/index.min.js"></script>
-    <script src="https://unpkg.com/@libp2p/websockets/dist/index.min.js"></script>
-    <script src="https://unpkg.com/@libp2p/kad-dht/dist/index.min.js"></script>
-    <script src="https://unpkg.com/@libp2p/identify/dist/index.min.js"></script>
-    <script src="https://unpkg.com/@libp2p/bootstrap/dist/index.min.js"></script>
-    <script src="https://unpkg.com/@libp2p/ping/dist/index.min.js"></script>
-    <script src="https://unpkg.com/@chainsafe/libp2p-gossipsub/dist/index.min.js"></script>
-    <script src="https://unpkg.com/libp2p/dist/index.min.js"></script>
-    <script src="https://unpkg.com/it-pipe/dist/index.min.js"></script>
-    <script src="https://unpkg.com/it-pb-stream/dist/index.min.js"></script>
-    <script type="text/javascript" charset="utf-8" src="js/blake2b.js"></script>
-    <script type="text/javascript" charset="utf-8" src="js/ed25519.js"></script>
-    <script type="text/javascript" charset="utf-8" src="js/http.js"></script>
-    <script type="text/javascript" charset="utf-8" src="js/fruzhin.js"></script>
+    <script type="module" src="js/fruzhin-lib.js"></script>
 </head>
 <body>
-<script>
-    var libp = undefined;
-
-    var getPeerId = () => {
-        return libp?.peerId;
-    }
-
-    var start = async (bootnodes) => {
-        console.log('Starting libp2p node');
-        console.log(bootnodes);
-        let test = {
-            addresses: {
-                listen: [],
-            },
-            transports: [
-                Libp2PWebsockets.webSockets()
-            ],
-            streamMuxers: [ChainsafeLibp2PYamux.yamux()],
-            connectionEncryption: [ChainsafeLibp2PNoise.noise()],
-            peerDiscovery: [
-                Libp2PBootstrap.bootstrap({
-                    list: bootnodes
-                })
-            ],
-            services: {
-                identify: Libp2PIdentify.identify(),
-                ping: Libp2PPing.ping(),
-                dht: Libp2PKadDht.kadDHT({protocol: "/dot/kad"}),
-                pubsub: ChainsafeLibp2PGossipsub.gossipsub(),
-            }
-        };
-
-        libp = await Libp2P.createLibp2p(test);
-        libp.start();
-
-        // libp.addEventListener('peer:discovery', (evt) => console.log('Discovered:', evt.detail.id.toString()))
-        // libp.addEventListener('peer:connect', (evt) => console.log('Connected:', evt.detail.toString()))
-        // libp.addEventListener('peer:disconnect', (evt) => console.log('Disconnected:', evt.detail.toString()))
-
-        // let pbStream = ItProtobufStream.pbStream;
-        //
-        // libp.handle('/dot/sync/warp', ({stream}) => {
-        //
-        //     console.log('Received a warp sync request');
-        //     stream.write('Hello from the other side');
-        //     stream.end();
-        // });
-    }
-    main();
+<script type="module" >
```

Comment:
```suggestion
<script >
```

the moduel type is obsolete here

Reply:
It doesn't find fruzhin.main() if type="module" is missing